### PR TITLE
147 app redesign v2

### LINKED ISSUE #147: Redesign the app
- [x] rearrange the navbar 
- [x] home, about etc
- [x] unify plots view 
- [x] loader (plots, dropbox), 
- [x] next stages blockers

### PR DESCRIPTION
**Pull request by azywot**
Closes #147 and #158
What's been done: plots view unified, loaders & next stages blockers added

Example:
obraz
Also unified the view of _Report_ stage:
obraz

✋ **I highly encourage you to go through the stages and lmk if there are anything to be improved regarding this issue.**

One I can spot for now is that despite wrapping the plot with a loader, there's ~1s during which it looks like this:
obraz
and after this 1s the loading sign appears (don't really know what to do with it, as this is some kind of a lag).

### CODE REVIEW

File: dashboard/pages/screening/stages/s2_outliers_purging.py [hunk 1]
```diff
@@ -81,19 +81,24 @@
         html.Div(
             className="d-flex flex-column gap-3 h-100",
             children=[
-                html.Div(
-                    className="overflow-auto mx-2 border border-3 rounded shadow bg-body-tertiary bg-primary flex-grow-1",
+                dcc.Loading(
```
Comment (zuzg) [inline dashboard/pages/screening/stages/s2_outliers_purging.py:84]: I think the loader is not needed when changing pages of bmg heatmaps (I would show it only on entering the stage)
The data is already loaded so the user wouldn't wait for anything anyway

File: dashboard/pages/screening/callbacks.py [hunk 2]
```diff
@@ -635,10 +679,17 @@ def on_json_generate_button_click(
 
 
 def register_callbacks(elements, file_storage):
+    # callback(
+    #     Output(elements["NEXT_BTN"], "disabled", allow_duplicate=True),
+    #     Input(elements["NEXT_BTN"], "n_clicks"),
+    #     prevent_initial_call=True,
+    # )(on_next_button_click)
```
Comment (zuzg) [inline dashboard/pages/screening/callbacks.py:686]: remove unnecessary comments or add explanation why it should stay

**Review by zuzg — COMMENTED** [review]
Looks good, I added some comments

File: dashboard/pages/screening/stages/s2_outliers_purging.py (hunk 1 above)
Comment (azywot) [reply dashboard/pages/screening/stages/s2_outliers_purging.py:84]: I see, the issue with `dcc.Loading` is that one cannot really specify if they want to have it loading only initially - there are some workarounds tho (eg. [this](https://community.plotly.com/t/loading-states-and-loading-component/19650/21)), so will try to incorporate it 🫡

**Comment by azywot** [comment]
Update: I made also an info icon function **(we still need to know where to put ℹ️)**
obraz

**Review by Tremirre — APPROVED** [review]
Lot's of good work I see
Code looks fine and the locally tested UI is very much improved
overall LGTM

as for the info icon - Imo we can place it relatively to the elements they relate to in the upper right corner, smthg like this:
image

**Comment by azywot** [comment]
> Lot's of good work I see Code looks fine and the locally tested UI is very much improved overall LGTM
> 
> as for the info icon - Imo we can place it relatively to the elements they relate to in the upper right corner, smthg like this: image

Getting there:
obraz
...but any help appreciated 🥴

**Review by zuzg — APPROVED** [review]

**Pull request merged**